Demo/Application/Features/PayPalWebCheckoutViewController.swift

```diff
@@ -159,7 +205,65 @@ class PayPalWebCheckoutViewController: PaymentButtonBaseViewController {
             self.completionBlock(nonce)
         }
     }
-    
+
+    // MARK: Edit FI flow
+
+    @objc func toggleErrorHandling(_ sender: UISwitch) {
+        let showErrorHandlingFields = sender.isOn
+        riskCorrelationIDLabel.isHidden = !showErrorHandlingFields
+        riskCorrelationIDTextField.isHidden = !showErrorHandlingFields
+    }
+
+    @objc func tappedPayPalEditVault(_ sender: UIButton) {
+        if errorHandlingToggle.isOn {
+            handlePayPalErrorHandlingEditVault(sender)
+        } else {
+            handlePayPalEditVault(sender)
+        }
+    }
+
+    private func handlePayPalEditVault(_ sender: UIButton) {
+        progressBlock("Tapped PayPal - Edit FI")
+        sender.setTitle("Processing...", for: .disabled)
+        sender.isEnabled = false
+
+
+        let vaultID = payPalVaultIDTextField.text ?? "+fZXfUn6nzR+M9661WGnCBfyPlIExIMPY2rS9AC2vmA="
+        let request = BTPayPalVaultEditRequest(editPayPalVaultID: vaultID)
+        payPalClient.edit(request) { editResult, error in
```

Comment:
super nit just to add a space here:
```suggestion
        let request = BTPayPalVaultEditRequest(editPayPalVaultID: vaultID)
        
        payPalClient.edit(request) { editResult, error in
```